info.yml

```diff
@@ -0,0 +1,18 @@
+exercises:
+  - name: intro
+    path: katas/intro.c
```

Comment:
couldn't we remove the folder name from here ? or maybe have a kata_folder variable somewhere in the yml ?

Reply:
I actually copied this structure from Rustlings, and honestly I think adding this is a lot of work for not so mush value.